### ISSUE TITLE
ruff check treats tomllib as not being part of the standard library

### ISSUE THREAD

**Issue by mgedmin** [astral-sh/ruff#27930]
### Summary

This is very similar to a closed bug #25100, except I'm not using any preview features or specifying --target-version py315.

Steps to reproduce:
- create a Python file that does
```py
import re
import tomllib

def somecode():
    # pretend code to avoid irrelevant unused import warnings
    re.compile()
    tomllib.loads()
```
- run `ruff check --isolated example.py`

Expected behavior:
- no errors

Actual behavior:
- ruff complains
```
I001 [*] Import block is un-sorted or un-formatted
 --> ruff_tomllib.py:1:1
  |
1 | / import re
2 | | import tomllib
  | |______________^
help: Organize imports
  |
1 | import re
2 +
3 | import tomllib
  |

Found 1 error.
[*] 1 fixable with the `--fix` option.
```

### Version

ruff 0.16.3

**Comment by ntBre** [comment]
I think you're running into the default [target-version](https://docs.astral.sh/ruff/settings/#target-version) setting, which is 3.10:

```console
❯ ruff check --isolated --show-settings | grep version
linter.unresolved_target_version = none
linter.per_file_target_version = {}
formatter.unresolved_target_version = 3.10
formatter.per_file_target_version = {}
analyze.target_version = 3.10
```

`tomllib` was only added to the stdlib in 3.11, so you'll have to set your `target-version` (or `requires-python` in a pyproject.toml) to 3.11 for Ruff to know it's valid.

**Comment by mgedmin** [comment]
Oh, makes sense!  I'll close the issue then.
